src/routes/views/components/charts/trendChart/trendChart.tsx

```diff
@@ -321,7 +322,7 @@ class TrendChartBase extends React.Component<TrendChartProps, State> {
         gutter={20}
         height={25}
         itemsPerRow={legendItemsPerRow}
-        name="legend"
+        name={chartName + '-legend'}
```

Comment:
Changing this name may break the interactive legend events. That is, when hovering over the legend, the associated data series is highlighted. Clicking on the legend item will hide / show the series.

Reply:
This name must match line 308 

https://github.com/project-koku/koku-ui/blob/0f570ecb28670fde733cdf917a7496f769a9060f/src/routes/views/components/charts/trendChart/trendChart.tsx#L308

```
     legendName: 'legend',
```

Reply:
good catch